src/ethereum/eth.js

```diff
@@ -135,7 +135,7 @@ export const eth = {
    * @return {object} The web3 provider
    */
   getWeb3Provider(httpProviderUrl = 'http://localhost:8545') {
-    return process.browser
+    return process.browser && typeof window !== 'undefined'
```

Comment:
This looks good! but do we know when `window` is `undefined` and `process.browser === true` ?

If it's common/unkown, maybe we can just drop the `process.browser` out of the conditional.

Reply:
Got it, since it is set by webpack we are aren't really checking the environment. Will remove it :+1: